### ISSUE TITLE
The resource list doesn't get updated on create [graphcool]

### ISSUE THREAD

**Issue by vysakh0** [marmelab/aor-graphql#34]
`/User`
For the first time, it will show "No results found"

`/User/create`
After I successfully create an user, and head back to `/User`, it still show "No results found". 

Clicking on `Refresh` button has no effect. Have to do a full reload. 

I even created a new record (User) manually in the graphcool backend, when going back and forth between routes, the new record does not show up  in `/User`.  But it works fine with update!

Am I doing something wrong? This is the case in two of my apps, please help!

**Comment by djhi** [comment]
Sorry we didn't found the time to answer you about this. I'll try to test that scenario again on the demo app tomorrow.

**Comment by djhi** [comment]
Ok I found the problem, that's because the Apollo client returns the list from its cache. I have to update the store with the newly created item.

**Comment by djhi** [comment]
Updating the store using the [`update` option](http://dev.apollodata.com/react/api-mutations.html#graphql-mutation-options-update) would be the best way to handle that, however I'll need some time to do it correctly.

In the mean time, I wanted to use the [`refetchQueries` option](http://dev.apollodata.com/react/api-mutations.html#graphql-mutation-options-refetchQueries) but I got `unknown query with name [queryName] asked to refetch` where queryName was indeed a correct query name (such as `allProducts` and `Product` for example.

The quickest workaround would be to set the `fetchPolicy` to `network-only` for all queries. To do so, you can use an undocumented yet feature:

When initializing the `aorClient`, pass it a `query` option:

```js
buildApolloClient({
    query: { fetchPolicy: 'network-only' },
})
```

**Comment by vysakh0** [comment]
@djhi thank you

**Comment by djhi** [comment]
Fixed by #41
